storagenode/storagenodedb/usedserials.go

```diff
@@ -13,46 +13,52 @@ import (
 	"storj.io/storj/storagenode/piecestore"
 )
 
+// ErrUsedSerials represents errors from the used serials database.
+var ErrUsedSerials = errs.Class("usedserials error")
+
 type usedSerials struct {
```

Comment:
again nit - could be renamed to `usedSerialsDB` possibly

Reply:
Done!